pkgs/tools/X11/wmutils-opt/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchFromGitHub, libxcb }:
+
+stdenv.mkDerivation rec {
+  name = "wmutils-opt-${version}";
+  version = "1.0";
+
+  src = fetchFromGitHub {
+    owner = "wmutils";
+    repo = "opt";
+    rev = "v${version}";
+    sha256 = "0mfhpja2l592i2a3j2l0lg65jhsdw1b57jnv2l0759ngpl4v82zk";
```

Comment:
Yeah... hash is now wrong:
```
unpacking source archive /tmp/nix-build-opt-v1.0-src.drv-0/v1.0.tar.gz
output path ‘/nix/store/swar2bmcm82243cb74i0mln8s89s9l8s-opt-v1.0-src’ has r:sha256 hash ‘0gd05qsir1lnzfrbnfh08qwsryz7arwj20f886nqh41m87yqaljz’ when ‘0mfhpja2l592i2a3j2l0lg65jhsdw1b57jnv2l0759ngpl4v82zk’ was expected
```

Put `0gd05qsir1lnzfrbnfh08qwsryz7arwj20f886nqh41m87yqaljz` in there.